### PR TITLE
cleaning files that are not needed

### PR DESCRIPTION
**Pull request by wooshrow**
Hi Premek,  I remove files and tests that are unlikely to be needed for SE integration. I leave some sources to be still red, they still have compile error, due to some dependency.

I want to propose some change in the architecture of SpaceEngEnvironment (or else the SocketEnv). Can we make this to extend eu.iv4xr.framework.mainConcepts.W3DEnvironment instead?

This allows a simpler interface to send commands, so that we dont have to keep constructing instances of AgentCommand.

### CODE REVIEW

File: src/main/java/agents/tactics/GoalLib.java [hunk 1]
```diff
@@ -132,21 +132,21 @@ public static GoalStructure entityInteracted(String entityId) {
         //the first goal is to navigate to the entity:
         var goal1 = 
         	  goal(String.format("This entity is in interaction distance: [%s]", entityId))
-        	  . toSolve((BeliefState belief) -> {
-        		  var e = (LabEntity) belief.worldmodel.getElement(entityId) ;
+        	  . toSolve((MyAgentBeliefState belief) -> {
+        		  var e = (MyGameXEntity) belief.worldmodel.getElement(entityId) ;
```
Comment (PremekPaska) [inline src/main/java/agents/tactics/GoalLib.java:136]: For now, just a question: What is the idea of the "MyGameX" prefix?

**Comment by wooshrow** [comment]
Hi Premek, it is a placeholder for another refactoring. Right know we 
have SeEntity. Ideally we want this SeEntity to extend the class 
WorldEntity from the framework. But I dont want to change this 
immediately, and propose that start a bit from the bottom by re-aligning 
SeEnvironment first, if that's ok with you guys.

--Wish

**Comment by PremekPaska** [comment]
OK, thanks for the clarification.

We have made a big cleanup and converted most of the project to Kotlin, so this pull request is no longer applicable. We will keep the branch as an inspiration. Closing the PR for now.

**Comment by PremekPaska** [comment]
There is also a split version of this PR here: https://github.com/iv4xr-project/iv4xrDemo-space-engineers/pull/10

**Pull request closed without merging**